hs_core/models.py

```diff
@@ -183,31 +183,50 @@ def can_view(self, request):
                          raises_exception=False)[1]
 
 
+# Build a JSON serializable object with user data
```

Comment:
I think it's worth settings up a serializer for these objects instead of only relying on a string.  We do this a lot in the rest api if you'd like more examples.

https://www.django-rest-framework.org/api-guide/serializers/

Reply:
This will be done in a further iteration once it is clear all the data that will be needed.